readthedocs/basic/signing-in.rst

```diff
@@ -127,15 +127,44 @@ with
 
 .. code-block:: python
 
-    TelegramClient('anon', api_id, api_hash, proxy=(socks.SOCKS5, '127.0.0.1', 4444))
+    TelegramClient('anon', api_id, api_hash, proxy=("socks5", '127.0.0.1', 4444))
 
 (of course, replacing the IP and port with the IP and port of the proxy).
 
 The ``proxy=`` argument should be a tuple, a list or a dict,
-consisting of parameters described `in PySocks usage`__.
+consisting of parameters described `in aiosocks usage`__.
 
-.. __: https://github.com/Anorov/PySocks#installation
-.. __: https://github.com/Anorov/PySocks#usage-1
+Example:
+
+* All of these are equal:
+
+.. code-block:: python
+
+    proxy = ('socks5', '1.1.1.1', 5555, True, 'foo', 'bar')
+    proxy = (socks.SOCKS5, '1.1.1.1', 5555, True, 'foo', 'bar')
+    proxy = (2, '1.1.1.1', 5555, True, 'foo', 'bar')
```

Comment:
Can we not encourage this? I would rather people not hardcode constants that then we need to keep track of.

Reply:
@Lonami 
See [this](https://github.com/LonamiWebs/Telethon/pull/1623/commits/32f60ac28acc870e6d36e2e1d9c4099589e85b77)